src/main/java/org/wmn4j/mir/discovery/Siatechf.java

```diff
@@ -0,0 +1,200 @@
+/*
+ * Copyright 2018 Otso Björklund.
```

Comment:
You have usually deleted these names from the project, did you mean to add this one? Not that I want to deny credits from you :smile:

Reply:
I removed that :)